### PR TITLE
Add support for multiple languages

### PR DESCRIPTION
**Pull request by ohgodnotagain**
1. `main.tex` is now `metadata.tex` stripped of any text that could be translated.
1. Every language will have its own main file, like `main_en.tex` with typesetting, chosen language, and the text which appears before any section.
1. For now, only English will be built and stored in the repo as a binary blob (which can be downloaded through readme).
1. Thanks to `po4a` introduced by @ohgodnotagain, there will be no changes to original tex files.  Instead, all the translated section files will live in `sections/translated/<lang>`.
1. All the translations will live in `translations/` directory independently of everything else.

### CODE REVIEW

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -75,6 +76,29 @@ Then, use the script:
 ./make_printable.sh
 ```
 
+### Translations 🌍
```
Comment (AFulgens) [inline README.md:79]: **chore:** All other sections start with the Emoji, so maybe we should replace this with "🌍 Translations"?

File: .github/workflows/build-and-upload.yaml [hunk 2]
```diff
@@ -46,21 +46,21 @@ jobs:
           python .github/insert_printable_hyperlinks.py
 
       - name: Include index page
-        run: sed -i "s@\\\include{sections/back_cover.tex}@\\\include{sections/index.tex}\\\include{sections/back_cover.tex}@g" main.tex
+        run: sed -i "s@\\\include{\\\sections/back_cover.tex}@\\\include{\\\sections/index.tex}\\\include{\\\sections/back_cover.tex}@g" metadata.tex
```
[the remark this answers is not in the document]
Comment (qwrtln) [reply .github/workflows/build-and-upload.yaml:49]: Nope. It's a variable now. Every language will have its own path to its own translations.

File: .github/workflows/build-rule-book.yaml [hunk 3]
```diff
@@ -16,6 +17,12 @@ on:
 jobs:
   build_rule_book:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language:
```
[the remark this answers is not in the document]
Comment (qwrtln) [reply .github/workflows/build-rule-book.yaml:23]: Ah yes, thanks

File: README.md (hunk 1 above)
Comment (qwrtln) [reply README.md:79]: Gosh, true, fixing!

**Pull request merged**